packaging/markers.py

```diff
@@ -13,8 +13,14 @@
 from pyparsing import Literal as L  # noqa
 
 from ._compat import string_types
+from ._typing import MYPY_CHECK_RUNNING
 from .specifiers import Specifier, InvalidSpecifier
 
+if MYPY_CHECK_RUNNING:  # pragma: no cover
+    from typing import Any, List, Tuple, Dict, Union, Optional, Callable
```

Comment:
Do we want to try and keep this alphabetized, or is that a fool's errand?

Reply:
I was trying to keep this alphabetized. I'll fix this but we should add [`isort`](https://pypi.org/project/isort/) if we really care about this.